tests/test_maintain.py

```diff
@@ -23,15 +25,18 @@ def test_create(client, key):
         client.delete("/metadata/" + key)
 
 
-def test_create_bad_guid(client):
-    bad_key = "upload"
+@pytest.mark.parametrize(
+    "forbidden_id",
+    config.FORBIDDEN_IDS,
```

Comment:
i think the unit test should not depend on the config here, because the requirement is that "upload" should be forbidden, not that a list of forbidden IDs can be customized - solved if the list is not a config anymore